src/dash-table/components/FilterFactory.tsx

```diff
@@ -55,7 +55,7 @@ export default class FilterFactory {
         }
 
         setFilter(R.map(
-            ([cId, filter]) => `${cId} ${filter}`,
+            ([cId, filter]) => `"${cId}" ${filter}`,
```

Comment:
Auto-quote the column id when generating the filter fragments from column filters